backend/src/main/java/com/greenfoxacademy/backend/controller/UserController.java

```diff
@@ -31,11 +34,10 @@ public class UserController {
    * This method registers a new user.
    *
    * @param registerUserDto the user to be registered
-   *
    * @return a response entity with the status code and the location of the new user
    */
-  @CrossOrigin (origins = "http://localhost:5173")
-  @PostMapping ("/register")
+  @CrossOrigin(origins = "http://localhost:5173")
```

Comment:
This will work for now, but we should change it later and make it configurable.

Reply:
You mean, environment variable?

Reply:
Yes, but we definitely should do it in another PR, it was just a heads-up, that this will change sooner or later.